lib/render/boards.html.erb

```diff
@@ -38,7 +38,7 @@
       <p class="title"><%=I18n.t it.title %></p>
       <p>
         <% if it.board.boom %>
-          <img src="<%= @boom_image %>" />
+          <img src="https://gobstones.runners.mumuki.io/boom.png" />
```

Comment:
This should probably be calculated, but I don't know how :slightly_frowning_face: